spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -137,6 +137,14 @@ def load_data(self, filename, ext):
                 blocking=True,
                 timeout=CALL_KERNEL_TIMEOUT).load_data(
                     filename, ext, overwrite=overwrite)
+        except ImportError as msg:
+            module = str(msg).split("'")[1]
+            msg = _("Spyder is unable to open the variable or object "
```

Comment:
```suggestion
            msg = _("Spyder is unable to open the file "
```

In this case users are trying to open a file, not a variable or an object.